From 660d30f424d9f48f9e179324a63d9dbe43334891 Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Fri, 1 Aug 2014 17:53:24 +0200
Subject: [PATCH] Disable telephony support until our service is in place

Don't spam the log with errors only because our device has telephony functionality but our
backend service isn't in place yet.

Upstream-Status: Inappropiate [configuration]

Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 src/ServiceApp.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/ServiceApp.cpp b/src/ServiceApp.cpp
index ef70096..a475f07 100644
--- a/src/ServiceApp.cpp
+++ b/src/ServiceApp.cpp
@@ -213,6 +213,7 @@ MojErr ActivityManagerApp::open()
 			boost::make_shared<SystemManagerProxy>(&m_client, m_am);
 		m_requirementManager->AddManager(smp);
 
+#if 0
                if(read_modem_present())
                {
                    boost::shared_ptr<TelephonyProxy> tp =
@@ -220,6 +221,7 @@ MojErr ActivityManagerApp::open()
                    m_requirementManager->AddManager(tp);
                }
 #endif
+#endif
 	} catch (...) {
 		return MojErrNoMem;
 	}
-- 
1.9.1

